vividus-tests/src/main/resources/story/integration/HTTP.story

```diff
@@ -38,15 +38,6 @@ Scenario: Verify HTTP cookies are cleared
 When I send HTTP GET to the relative URL '/cookies'
 Then JSON element by JSON path `$.cookies` is equal to `{}`
 
-Scenario: I wait for response code $responseCode for $duration duration retrying $retryTimes times $stepsToExecute
```

Comment:
Any reasons to remove the scenario?

Reply:
> Any reasons to remove the scenario?

It seemed that this scenario immediately returns 200 status code and does not use the logic of retries. But I ran the scenario - the retries are used. Ok, returned the old scenario test.

Reply:
I like new scenario, it looks more accurate in terms of validations

@draker94 could you please restore your scenario?

@ikalinin1 having new scenario, the old one will be redundant, thus it should be removed

Reply:
would you be so kind to elaborate "more accurate"

Reply:
Also, there is no reason to keep the logic in test site then, so please consider removal or create a ticket for this

Reply:
Okay, but the new scenario doesn't validate if anything happened in the step, previous one validated status code after the step

Reply:
reasonable, makes sense to add this validation